src/settings/utils/code-formatters.js

```diff
@@ -0,0 +1,37 @@
+import { find } from 'lodash';
```

Comment:
Good idea moving all these to utils. I wonder about prepend the file name with location to make it `location-code-formatters` in order to add more context. What do you think about that?

Reply:
@VictorSoroka1 thanks for you comment. I renamed the file.